addons/mail/static/src/messaging/component/attachment_list/attachment_list.xml

```diff
@@ -3,11 +3,11 @@
 
     <t t-name="mail.messaging.component.AttachmentList" owl="1">
         <div class="o_AttachmentList">
-            <t t-foreach="props.attachments" t-as="attachment" t-key="attachment">
+            <t t-foreach="props.attachmentLocalIds" t-as="attachmentLocalId" t-key="attachmentLocalId">
```

Comment:
props in template, should we use attachments entity instead?

Reply:
I wouldn't because this component doesn't have `useStore`.